### ISSUE TITLE
tab_spanner_delim fails with uninformative error when delim occurs more than once

### ISSUE THREAD

**Issue by LukasWallrich** [rstudio/gt#685]
When the delim symbol used in `tab_spanner_delim()` occurs more than once, execution fails with Error in FUN(X[[i]], ...) : invalid negative subscript in get1index <real>. In the absence of an option to stack spanners, I am not sure what the desired behaviour should be ... I would probably split at the first delim and raise a warning, but an informative error message would also be ok. As it stands, however, I found this unnecessarily hard to debug, especially compared to other parts of this great package.

```r
iris %>% group_by(Species) %>% summarise(across(everything(), ~mean(.), .names = "M.{col}"),
                                         across(everything(), ~sd(.), .names = "SD.{col}")) %>% gt::gt() %>% gt::tab_spanner_delim(".")
```

**Comment by rich-iannone** [comment]
Thanks for letting me know about this. I hope to fix this bug soon.

**Comment by rich-iannone** [comment]
This is actually now fixed in gt `0.3.0` (split occurs on the `last` delimiter mark by default).
